test_accounts/appeals_apis/appeals_status_test_accounts.md

```diff
@@ -0,0 +1,45 @@
+# Sandbox Test Accounts for Appeals Status API (v1)
+
+This document provides test user accounts for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current). The API will return synthetic VA records for all test accounts.
+
+**Do not modify any of the attributes for these test accounts, including passwords.** If you need to use one of these accounts for a different use case, [contact us](https://developer.va.gov/support/contact-us) on our developer portal.
+
+## Authentication and Authorization
+
+The authentication model for the [Appeals Status API (v1)](https://developer.va.gov/explore/api/appeals-status/docs?version=current) uses OAuth 2.0/OpenID Connect. The following authorization models are supported:
+- [Authorization code flow](https://developer.va.gov/explore/api/appeals-status/authorization-code)
+- [Client Credentials Grant (CCG)](https://developer.va.gov/explore/api/appeals-status/client-credentials)
+
+### Logging in with ID.me
```

Comment:
Addressing your comment about the `appeals_api` and `appealsAPI` directories, I agree that could be confusing. Given that `appealsAPI` hasn't been updated in 5 years, I do also wonder if it's still in use. With that said, Casey and I discussed the directory structure and agreed that it makes more sense to have the test account page directly  under `test_accounts` without an `appeals_api` subdirectory since all other API test account pages live directly under `test_accounts` and "Appeals APIs" is really a grouping that we use on our end, not a grouping that consumers would necessary use. (The developer portal doesn't group the Appeals APIs together in any way, for example.)

I went ahead and removed the "Logging in with..." sections in my latest changes given the public platform's plan to move this information to the public portal, and replaced those sections with a new sentence under the "Authentication and Authorization" heading:
`Passwords for all test accounts can be found on the [developer portal](link-will-go-here-once-page-created).`

It's a broken link for now. I'm thinking we won't merge this PR until that developer portal page has been created and the broken link has been updated.